packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -122,7 +122,26 @@ export const styles = theme => {
         height: 17,
       },
     },
-    /* Class applied to the root element to trigger JSS nested styles if `reverse={true}` . */
+    /* Class applied to the thumb element if custom thumb icon provided. */
+    thumbIconWrapper: {
+      backgroundColor: 'transparent',
+      width: 17,
+      height: 17,
+      '&$activated': {
+        width: 22,
+        height: 22,
+        transition: 'none',
```

Comment:
The thumb should grow and shrink.

Reply:
the thumb does grow and shrink.

Reply:
![4azq7c9svl](https://user-images.githubusercontent.com/16651811/45377826-2409f180-b615-11e8-8031-41818980bc2e.gif)

As you can see in the demo, the icon does grow and shrink.

Reply:
I had some bloated styling, which was redundant. I removed it.

Reply:
That seems to have fixed it in Firefox